screens/NewPostImage.tsx

```diff
@@ -82,6 +127,39 @@ export function NewPostImage({ navigation }) {
     }
   };
 
+  const takePhoto = async () => {
+    const cameraPermission = await checkCameraPermission();
+    if (!cameraPermission) {
+      Alert.alert(
+        "Permission Required",
+        "Please enable camera permissions to use this feature.",
+        [
+          {
+            text: "Cancel",
+            style: "cancel",
+          },
+          {
+            text: "Settings",
+            onPress: openAppSettings,
+          },
+        ]
+      );
+      return;
+    }
+
+    const result = await ImagePicker.launchCameraAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.Images,
+      quality: 1,
+    });
+
+    console.log(JSON.stringify(result))
```

Comment:
Let's remove this console log. I want to keep the codebase as clean as possible with logs.